CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased](https://github.com/OpenLineage/OpenLineage/compare/0.6.2...HEAD)
 ### Added
+* Airflow integration uses [new TaskInstance listener API](https://github.com/apache/airflow/blob/main/docs/apache-airflow/listeners.rst) for Airflow 2.3+ [@mobuchowski](https://github.com/mobuchowski)
```

Comment:
The following doc section might need updating too

https://github.com/OpenLineage/OpenLineage/blob/f0b8b5ce30c5f82971e25674142e54ae16ac312e/integration/airflow/docs/task-lifecycle.md#openlineage-interaction

Reply:
Done.